Sage/carbon
# docs: updated favicon

### Proposed behaviour

- Display the Carbon logo as the favicon in the desktop browser tab [FE-5627](https://jira.sage.com/browse/FE-5627)
- Display the Carbon logo on the Home screen if the user selects **Add to Home screen** on a mobile device
- Changes the value of `brandTitle`. which appears when you mouse over the logo on the top left of the screen to "Carbon by Sage".
- Changes spelling of "Github" to "GitHub" on Welcome page
- Other minor capitalization and punctuation changes

### Current behaviour

- The current favicon is the default Storybook logo 
![image](https://user-images.githubusercontent.com/83212011/214024911-ad99c32a-fdb5-4b02-940a-c941def17a33.png)

- `brandTitle` is currently "Sage DS by Carbon"

### Checklist


#### QA


### Testing instructions

Run Carbon locally and check that the favicon displayed in the browser tab is Carbon, not Storybook.

![image](https://user-images.githubusercontent.com/83212011/214031330-2d60f652-756a-47be-9e84-c46649a3883b.png)

Mouse over the Carbon logo in the top left of the screen to display the correct Alt text - "Carbon by Sage"

Use the network IP address and access the Carbon site from a mobile device. Select **Add to Home Screen** and check that the Carbon logo appears on the device's home screen.
